src/server/pps/cmds/cmds.go

```diff
@@ -1693,25 +1693,26 @@ func Cmds(mainCtx context.Context, pachCtx *config.Context, pachctlCfg *pachctl.
 	commands = append(commands, cmdutil.CreateAliases(deleteDefaults, "delete defaults"))
 
 	updateDefaults := &cobra.Command{
-		Use:   "{{alias}} [--cluster]",
+		Use:   "{{alias}} [--cluster | --project PROJECT]",
 		Short: "Update defaults.",
-		Long:  "Update defaults.",
+		Long:  "Update cluster or project defaults.",
 		Run: cmdutil.RunFixedArgs(0, func(args []string) error {
+			rc, err := fileIndicatorToReadCloser(pathname)
+			if err != nil {
+				return errors.Wrapf(err, "could not open path %q for reading", pathname)
+			}
 			if cluster {
-				rc, err := fileIndicatorToReadCloser(pathname)
-				if err != nil {
-					return errors.Wrapf(err, "could not open path %q for reading", pathname)
-				}
 				return setClusterDefaults(mainCtx, pachctlCfg, rc, regenerate, reprocess, dryRun)
 			}
-			return errors.New("--cluster must be specified")
+			return setProjectDefaults(mainCtx, pachctlCfg, rc, regenerate, reprocess, dryRun)
```

Comment:
What if project wasn't set either? Should we explicitly handle this and error?

Reply:
Yup, fixed now.